package.json

```diff
@@ -38,7 +39,7 @@
     "@nestjs/platform-express": "10.0.5",
     "@nestjs/swagger": "7.1.1",
     "@prisma/client": "5.0.0",
-    "bcrypt": "5.1.0",
+    "bcryptjs": "2.4.3",
```

Comment:
I remember you were mentioning an issue with `bcrypt` - could you elaborate here what was the issue and why did we change it to `bcryptjs`. If I remember correctly it was connected to using the `alpine` linux distribution

Reply:
On `npx run build` bcrypt_module could not be generated. People suggested using `bcryptjs` or installing `gyp` (https://github.com/kelektiv/node.bcrypt.js/issues/800). Also, I don't remember where but people pointed out that it is due to the fact `bcrypt` uses C++ (https://codeforgeek.com/bcrypt-vs-bcryptjs/) where bcryptjs uses plan JS.